projects/aca-playwright-shared/src/page-objects/components/dialogs/content-node-selector-dialog.ts

```diff
@@ -40,13 +40,13 @@ export class ContentNodeSelectorDialog extends BaseComponent {
   private getRowByName = (name: string | number): Locator => this.getChild(`adf-datatable-row`, { hasText: name.toString() });
   getDialogTitle = (text: string) => this.getChild('[data-automation-id="content-node-selector-title"]', { hasText: text });
   getBreadcrumb = (text: string) => this.getChild('[data-automation-id="current-folder"]', { hasText: text });
-  getFolderIcon = this.getChild('mat-icon[role="img"]', { hasText: "folder" });
+  getFolderIcon = this.getChild('mat-icon[role="img"]', { hasText: 'folder' });
```

Comment:
this selector will be flaky af, can it be more specific, and without the "role" attribute which is the accessibility, not the testing

Reply:
I will check that but many locators are missing test-automation-id and adding them is not a simple process sadly